Do we really need https://raw.github.com/plone/plone.app.robotframework/master/versions.cfg in buildout.cfg's extend section?

We already extend http://dist.plone.org/release/4.3-latest/versions.cfg which has a pretty decent pinning for robots:

```cfg
# Robot Testing
plone.app.robotframework = 1.0
robotframework = 3.0
robotframework-debuglibrary = 0.4
robotframework-ride = 1.5.2.1
robotframework-selenium2library = 1.7.4
robotframework-selenium2screenshots = 0.7.0
robotsuite = 1.7.0
selenium = 2.53.5
sphinx-rtd-theme = 0.1.5
sphinxcontrib-robotframework = 0.5.1
```

Why exactly do we need to extend https://raw.github.com/plone/plone.app.robotframework/master/versions.cfg again?

https://travis-ci.org/collective/collective.cover/builds/170107306

In 4.2: WebDriverException: Message: 'geckodriver' executable needs to be in PATH. 

In 4.3 with jquery: AssertionError: Element 'xpath=.//a[@class='calendar-tile-next'][@data-month='11'][@data-year='2016']' did not appear in 30 seconds

--- a/buildout.cfg
+++ b/buildout.cfg
@@ -2,7 +2,6 @@
 extends =
     https://raw.github.com/collective/buildout.plonetest/master/test-4.3.x.cfg
     https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
-    https://raw.github.com/plone/plone.app.robotframework/master/versions.cfg
     versions-4.3.x.cfg
 
 package-name = collective.cover

--- a/versions-4.2.x.cfg
+++ b/versions-4.2.x.cfg
@@ -4,6 +4,18 @@ test-eggs =
     plone.app.referenceablebehavior
 # https://github.com/testing-cabal/mock/issues/261
     funcsigs
+    
+
+extends = 
+# When trying to remove plone.app.robotframework pins (check
+# https://github.com/collective/collective.cover/issues/672)
+# We realized that http://dist.plone.org/release/4.2-latest/versions.cfg
+# doesn't pin any of this infrastructure, thus giving
+# "WebDriverException: Message: 'geckodriver' executable needs to be in PATH."
+# error. So we can remove this pin from 4.3 onwards since
+# http://dist.plone.org/release/4.3-latest/versions.cfg pins robots
+# infrastructure but 4.2 still needs it.
+    https://raw.github.com/plone/plone.app.robotframework/master/versions.cfg
 
 [versions]
 collective.js.bootstrap = 2.3.1.1

